Make build process free of warnings and errors

_The following comes from following the [build instructions](https://github.com/ForkAwesome/Fork-Awesome#building-fork-awesome) on Ubuntu 20.10._

## Building the fonts
`make -C src/icons` succeeds, with _"Using the last argument as keyword parameters is deprecated"_ warnings. These seem to arise from using [outdated ruby gems](https://bloggie.io/@kinopyo/how-to-fix-ruby-2-7-warning-using-the-last-argument-as-keyword-parameters-is-deprecated). Terminal dump:

```
$ make -C src/icons
make: Entering directory '/home/doug/ext/Fork-Awesome/src/icons'
Cleaning up SVG files...
npx svgo --quiet --pretty --config=svgo_config.json --folder=svg
Compiling Icons into a ForkAwesome fonts...
bundle exec fontcustom compile
/var/lib/gems/2.7.0/gems/json-1.8.6/lib/json/common.rb:155: warning: Using the last argument as keyword parameters is deprecated
/var/lib/gems/2.7.0/gems/json-1.8.6/lib/json/common.rb:155: warning: Using the last argument as keyword parameters is deprecated
      create  forkawesome
/var/lib/gems/2.7.0/gems/json-1.8.6/lib/json/common.rb:155: warning: Using the last argument as keyword parameters is deprecated
      create  forkawesome/forkawesome.ttf
              forkawesome/forkawesome.svg
              forkawesome/forkawesome.woff
              forkawesome/forkawesome.eot
              forkawesome/forkawesome.woff2
/var/lib/gems/2.7.0/gems/json-1.8.6/lib/json/common.rb:155: warning: Using the last argument as keyword parameters is deprecated
      create  forkawesome/forkawesome.css
              forkawesome/forkawesome-preview.html
/var/lib/gems/2.7.0/gems/json-1.8.6/lib/json/common.rb:155: warning: Using the last argument as keyword parameters is deprecated
Copying builds to root folder (package release)...
cp ./forkawesome/forkawesome.eot ../../fonts/forkawesome-webfont.eot
cp ./forkawesome/forkawesome.svg ../../fonts/forkawesome-webfont.svg
cp ./forkawesome/forkawesome.ttf ../../fonts/forkawesome-webfont.ttf
cp ./forkawesome/forkawesome.woff ../../fonts/forkawesome-webfont.woff
cp ./forkawesome/forkawesome.woff2 ../../fonts/forkawesome-webfont.woff2
done.
make: Leaving directory '/home/doug/ext/Fork-Awesome/src/icons'
```

## Building the website
`npm run build`, which aliases to `bundle exec jekyll build` within `package.json`, **fails** with:

```
lessc --clean-css="--compatibility=ie8" --source-map assets/fork-awesome/css/fork-awesome.css assets/fork-awesome/css/fork-awesome.min.css
undefinedError: Options have been provided but the plugin index.js does not support any options.
```

More specifically, the website html files are built, but there is no styling. Additionally, there is tsunami of the warnings mentioned above. Terminal dump:

```
...
...
...
/var/lib/gems/2.7.0/gems/jekyll-3.7.4/lib/jekyll/tags/include.rb:191: warning: Using the last argument as keyword parameters is deprecated
/var/lib/gems/2.7.0/gems/jekyll-3.7.4/lib/jekyll/tags/include.rb:191: warning: Using the last argument as keyword parameters is deprecated
Cleaning build dirs
rm -rf ./less/
rm -rf ./scss/
Compiling Less files
lessc assets/fork-awesome/less/fork-awesome.less assets/fork-awesome/css/fork-awesome.css
lessc --clean-css="--compatibility=ie8" --source-map assets/fork-awesome/css/fork-awesome.css assets/fork-awesome/css/fork-awesome.min.css
undefinedError: Options have been provided but the plugin index.js does not support any options.


make: *** [Makefile:30: build] Error 1
                    done in 7.875 seconds.
 Auto-regeneration: disabled. Use --watch to enable.
```

--- a/package.json
+++ b/package.json
@@ -32,8 +32,8 @@
     "node": ">=0.10.3"
   },
   "scripts": {
-    "build": "bundle exec jekyll build",
-    "dev": "bundle exec jekyll serve -w",
+    "build": "RUBYOPT='-W0' bundle exec jekyll build",
+    "dev": "RUBYOPT='-W0' bundle exec jekyll serve -w",
     "contributors:add": "all-contributors add",
     "contributors:generate": "all-contributors generate"
   },

